tests/product/tests/conftest.py

```diff
@@ -199,6 +204,9 @@ def pytest_generate_tests(metafunc):
         raise ValueError(f"Params for function {metafunc.function} are not registered.")
 
     test_range = metafunc.config.getoption("range")
+    if "eks" in metafunc.config.getoption("markexpr", default=None):
+        test_range = ".."
```

Comment:
then why do we set an empty string in the workflow for the test ranges?

Reply:
nice catch, thanks. It is not required. If range not provided the default option is used.